src/libraries/Common/src/System/Security/Cryptography/ECOpenSsl.cs

```diff
@@ -86,12 +86,12 @@ internal int GenerateKey(ECCurve curve)
 
             if (curve.IsNamed)
             {
-                string oid = null;
+                string oid;
                 // Use oid Value first if present, otherwise FriendlyName because Oid maintains a hard-coded
                 // cache that may have different casing for FriendlyNames than OpenSsl
-                oid = !string.IsNullOrEmpty(curve.Oid.Value) ? curve.Oid.Value : curve.Oid.FriendlyName;
+                oid = !string.IsNullOrEmpty(curve.Oid.Value) ? curve.Oid.Value : curve.Oid.FriendlyName!;
```

Comment:
Nit: is there a reason we define `string oid;` on its own line?

Reply:
not looks like